src/sdl/main.cpp

```diff
@@ -40,7 +46,7 @@ static void check_video_driver(const char *name)
 }
 
 
-int main(int argc, char *argv[]) {
+int main(int /* argc */, char ** /* argv */) {
```

Comment:
This project requires C++17 as a minimum, which means you have access to `[[maybe_unused]]` here. Since this project emphasizes best-practices, I think it makes more sense here to use the attribute rather than comment out the variable names.
```suggestion
int main([[maybe_unused]] int argc, [[maybe_unused]] char ** argv) {
```

Reply:
yes, and no.

In this case, the parameters are never used.

I would prefer to use
```c++
int main() {
```

Reply:
> I would prefer to use
> 
> ```c++
> int main() {
> ```
I agree, that looks much better.

Reply:
ok

Reply:
this break the `travis` build with `MVC`?

Reply:
fixed